VM/rel2vm_params.py

```diff
@@ -756,6 +778,16 @@ def optimise_vm_params(
 
     # modified sim time
     vm_corners = np.asarray([c1, c2, c3, c4])
+
+    if faultprop.Mw >= 7.2:
+        ds_multiplier = 0.9
+    elif faultprop.Mw > 6:
+        ds_multiplier = -0.3 / 1.3 * (faultprop.Mw - 6) + 1.2
+    else:
+        ds_multiplier = 1.2
+
+    print(f"{faultprop.Mw} {ds_multiplier}")
+
```

Comment:
Where has this come from???

Reply:
Ouch, a quick-and-dirty fix meant to be another temp branch sneaked in.....

Reply:
Removed!